Source/Mosa.Kernel.BareMetal.x86/PageTable.cs

```diff
@@ -16,30 +16,43 @@ internal static class PageTable
 
 		public static void Setup()
 		{
+			//Console.WriteLine("Mosa.Kernel.BareMetal.x86.PageTable.Setup:Enter");
```

Comment:
Looks like a whole lot of debug messages have been left in, is this intentional?

Reply:
Yeah, still debugging this.